txstratum/prometheus.py

```diff
@@ -43,7 +91,6 @@ def collect_metrics(manager: 'TxMiningManager') -> MetricData:
     return MetricData(
         miners_count=len(manager.miners),
         total_hashrate_ghs=manager.get_total_hashrate_ghs(),
-        txs_solved=manager.txs_solved,
```

Comment:
I was thinking about keeping the global txs solved count. Is it bad?

Reply:
We will have the total count from the Histogram metric, this one would be redundant.

Reply:
Ah, ok. I thought the new one was just aggregating by miner, so the total count could be different, given that two different miners could solve the same tx.

But if we have the two different metrics, fine by me

Reply:
As we discussed privatly, this is fine